queries/procedures/create_unicode_fix_v8.sql

```diff
@@ -0,0 +1,41 @@
+DROP PROCEDURE IF EXISTS CleanUnicodeEscapes;
+
+CREATE PROCEDURE CleanUnicodeEscapes(IN table_name VARCHAR(64), IN column_name VARCHAR(64))
+BEGIN
+    DECLARE done INT DEFAULT FALSE;
+    DECLARE changes_made INT DEFAULT 1;
+    DECLARE sql_statement TEXT;
+    
+    -- Keep running until no more changes are made
+    WHILE changes_made > 0 DO
+        SET changes_made = 0;
+        
+        -- Remove any occurrence of \u0000 with any number of preceding backslashes
+        SET sql_statement = CONCAT(
+            'UPDATE `', table_name, '` ',
+            'SET `', column_name, '` = REGEXP_REPLACE(`', column_name, '`, ''\\\\\\\\+u0000'', '''') ',
+            'WHERE `', column_name, '` REGEXP ''\\\\\\\\+u0000'''
```

Comment:
So this regex is for 2 or more backslashes right? I would expect a total of 4 backslashes for that? Why do we need 8 backslashes?

Reply:
Well that's a escaping hell indeed, compounded with regexp and sql escaping for the statement. This ends up to actually interpreted as `\\+u0000` which should find `\u0000` and more backslashes.

Reply:
The main reason I'm trying to steer away from Go code is keeping all the queries individually usable. Let's see if this query work, if not probably we need to do more (probably slower) intensive checks in the app layer.

On the tests, I'm with you and I wish we had those mindset from the get go. I think we never imagined `migration-assist` ending up being a very comprehensive tool or even being an essential part of the workflow 🤷